src/js/components/DialogsComponent.jsx

```diff
@@ -40,7 +41,9 @@ var DialogsComponent = React.createClass({
       DialogEvents.CONFIRM_SHOW,
       this.onDialogConfirmShow
     );
-    DialogStore.removeListener(DialogEvents.CONFIRM_DISMISS, this.onDialogClose);
+    DialogStore.removeListener(
+      DialogEvents.CONFIRM_DISMISS, this.onDialogClose
```

Comment:
For line breaking arguments I would prefer one of the two:

```
DialogStore.removeListener(DialogEvents.CONFIRM_DISMISS, 
  this.onDialogClose
);
```

or

```
DialogStore.removeListener(
  DialogEvents.CONFIRM_DISMISS, 
  this.onDialogClose
);
```

with a slight preference for the former when the first argument fits. What do you think, @aldipower and @philipnrmn ?
(Nope, airbnb's guide does not cover this ;) )